jsapp/scss/components/_kobo.map.scss

```diff
@@ -399,6 +399,28 @@ body .leaflet-control-layers,
   }
 }
 
+.map-no-geopoint{
+  text-align:center;
+  margin:auto;
+  padding:10px;
+  font-weight:bold;
+}
+.map-no-geopoint-wrapper{
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width:100%;
+}
+.map-transparent-background{
+  height: 100%;
+  position: absolute;
+  width: 100%;
+  background-color: grey;
+  z-index: 401;
```

Comment:
I think this should be `z-index: 1001;` - so the message covers all the UI. If there is no data, there is no point in user interacting with it.

Reply:
Then you will need to add `z-index: 0;` to `.form-view--map`, so the background is contained within the map div.